### PR TITLE
Haplo - check if chromosome is valid 

### PR DESCRIPTION
**Pull request by dglemos**

### CODE REVIEW

File: modules/Bio/EnsEMBL/VEP/Haplo/Parser/VCF.pm [hunk 1]
```diff
@@ -191,4 +193,35 @@ sub create_VariationFeatures {
   }];
 }
 
+=head2 validade_chr
```
Comment (helensch) [inline modules/Bio/EnsEMBL/VEP/Haplo/Parser/VCF.pm]: Type in header

File: modules/Bio/EnsEMBL/VEP/Haplo/Parser/VCF.pm [hunk 2]
```diff
@@ -191,4 +193,35 @@ sub create_VariationFeatures {
...
+
+  Arg 1      : hash
+  Example    : my $vf_hashes = $parser->validade_chr($vf);
```
Comment (helensch) [inline modules/Bio/EnsEMBL/VEP/Haplo/Parser/VCF.pm]: Typo in example

File: modules/Bio/EnsEMBL/VEP/Haplo/Parser/VCF.pm (hunk 1 above)
Comment (dglemos) [reply modules/Bio/EnsEMBL/VEP/Haplo/Parser/VCF.pm]: That's right. Thanks.

**Comment by ens-lgil** [comment]
A general question: would it be useful for VEP as well ?
I mean, should we move the new method `validate_chr` in `Bio::EnsEMBL::VEP::Parser` or `Bio::EnsEMBL::VEP::Parser::VCF` ?

**Comment by at7** [comment]
merged with master and release/97

**Pull request merged**